### ISSUE TITLE
[Bug] Remove #boardcomponents, #monsters, and #shop, is no longer available in editor

### ISSUE THREAD

**Issue by antontimmermans** [NPBruce/valkyrie#1498]
The editor does not provide the option to remove #monsters, #boardcomponents, and #shop.
![image](https://user-images.githubusercontent.com/42072372/120064978-53841500-c06f-11eb-948b-8cae3cae8c38.png)

The description of the use of these variables can be found on: https://github.com/NPBruce/valkyrie/wiki/QuestIniEvent#remove

I have used this in older scenarios and must have had access in the editor to these items (or else I would not have been able to use this). The fact that they are no longer available in 2.5.6 sounds like regression. I cannot say in which version this was still working.

### Scenario
-any -

### Steps to reproduce the behavior:
1. Create a new scenario
2. add any new tile
3. add any spawn
4. create a new event, click on 'remove': -->specific tile and spawn show up, but not the #<variables>

### Expected behavior
When in the editor in an event items are removed, the #<variables> (#monsters, #boardcomponents, #shop) must be selectable.
Note: When #monsters is removed, It should NOT trigger any of the removed monsters 'defeated' event.

### Valkyrie Version
2.5.6

### Desktop
 - OS: Windows
 - Browser Chrome
 - Version 10

**Comment by NicolaZee** [comment]
This is an essential fix for players who do not want to manually edit the scenario text files.

**Comment by malkavk** [comment]
I use Valkyrie in Portuguese. I'm building (making? writing?) a scenario where the default language is Portuguese.
I changed Valkyrie's language to English and this problem happened.
I changed the Valkyrie language back to Portuguese and these components were available again.

**Comment by antontimmermans** [comment]
This issue appears to be the same as #1480

**Comment by antontimmermans** [comment]
I hope this problem gets solved in the next version of Valkyrie, but until it does this is a workaround:

**IMPORTANT:** you will need to use a plain text editor (free source code editor like "Atom" or "Notepad++")

**1:** go to the event where you want to remove #boardcomponents, #monsters, or #shop

**2:** in that event, Click on to + next to 'Remove Components' and you can select a component from the list to remove. Just select any component (one you can easily recognize, like a tile for example).
![image](https://user-images.githubusercontent.com/42072372/136045306-59f78093-1f89-40fa-8e6d-3c9f64c6a797.png)

**3:** Save your scenario (and check in which *.ini file your event is stored ... by default: events.ini).you can find it at the bottom of the event:
![image](https://user-images.githubusercontent.com/42072372/136045710-c2afffd2-07c5-41f3-aad0-4a40c3cd4a3a.png)

**4:** Open the *.ini file in the sourcecode editor and find your event.
it will look something like this:
```
[EventEndCleanUp]
display=false
buttons=1
event1=EventEndEpilogue
remove=Tile0
```
**5:** now all you need to do is to replace the component on the remove line with any of the special items. It will then look like this for example:
```
[EventEndCleanUp]
display=false
buttons=1
event1=EventEndEpilogue
remove=#monsters #boardcomponents
```
**6:** save the file, and reload your scenario in Valkyrie where it will look like this: 
![image](https://user-images.githubusercontent.com/42072372/136044421-e625b672-50f8-4efd-9ce4-6e5f93ecb1f8.png)


ENJOY!

**Comment by mayjak** [comment]
Fixed in coming 2.5.8 release.

**Comment by antontimmermans** [comment]
Tested in 2.5.8a;
When you click on [+] at Remove Components (see image above).
- the 'Special' item appears in the selectable options
- the items #shop #boardcomponents #monsters are selectable.
the removal of the selected items appears to work as normal during gameplay (not: I did not test #shop) 

**Side effect:**
When you click on [+] at Add Components (see image above), you can also select Special and all items mentioned above. Valkyrie will crash when you reach the event in which you add these 'special' items. 

Typically #boardcomponents #monsters should not be added in this way. I can't say for #shop since I never used it, but I assume it is also not the way to add one. SO I think the 'Special' items may only be visible with Remove Components.
